manim/mobject/geometry.py

```diff
@@ -406,6 +428,41 @@ def __init__(
         )
 
     def surround(self, mobject, dim_to_match=0, stretch=False, buffer_factor=1.2):
+        """Modifies a circle so that it surrounds a given mobject.
+
+        Parameters
+        ----------
+        mobject : :class:`~.Mobject`
+            The mobject that the circle will be surrounding
+        buffer_factor :  Optional[:class:`float`]
+            The distance between the mobjects. A buffer_factor < 1 makes the circle smaller than the mobject.
```

Comment:
This isn't actually the distance between the mobjects, but a scaling factor. I think however that a  buffer distance would make more sense. You might want to create an Issue for that.

Reply:
Thanks, I fixed the mistake. I think a scaling factor makes sense for `surround` since a specific `buff` would be hard to define for asymmetric mobjects